pkg/log/config.go

```diff
@@ -52,6 +53,8 @@ func setOutput(filename string) {
 	filename = strings.Trim(filename, " ")
 	if filename == "/dev/null" {
 		logrus.SetOutput(ioutil.Discard)
+	} else if filename == "-" {
```

Comment:
Should we use the condition `filename == ""` instead of `filename == "-"` to judge if log to stdout?